lib/ace/editor.js

```diff
@@ -502,6 +502,29 @@ var Editor = function(renderer, session) {
         this.setOption("fontSize", size);
     };
 
+    this.$showCoordinate = function() {
+        if (this.session.$bracketHighlight) {
```

Comment:
is it just copy-paste? Are you sure that this `if` block is really required here?